scripts/install_packages.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -o errexit \
+    -o pipefail \
+    -o nounset
+
+tlmgr update --self
+tlmgr install \
+    datetime \
+    fmtcount \
+    enumitem \
+    titlesec \
+    siunitx \
+    tikzpagenodes \
+    ifoddpage \
+    lipsum \
+    ragged2e \
+    footmisc \
+    tocloft \
+    minitoc \
+    chappg \
+    tcolorbox \
+    environ \
+    luatex85 \
+    changebar
```

Comment:
For future reference: what determines if a package needs to be explicitly listed here?

Reply:
I simply added packages based on the errors I got until there were no more errors.

I think the strategy should work both ways : we should remove everything we can from this list until it crashes.